src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -240,11 +247,11 @@ public synchronized static void parse(DLNAMediaInfo media, InputFile inputFile,
 						}
 
 						value = MI.Get(audio, i, "BitDepth");
-						if (!value.isEmpty()) {
-							try {
+						if (isNotBlank(value) || !value.contains("Bit")) {
+							if (!value.contains("/")) {
 								currentAudioTrack.setBitsperSample(Integer.parseInt(value));
-							} catch (NumberFormatException nfe) {
```

Comment:
@Sami32 I understand, but this now is a much bigger bug. You can read how it works here:
https://docs.oracle.com/javase/tutorial/essential/exceptions/handling.html